generic-ec/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "generic-ec"
-version = "0.1.4"
+version = "0.1.5"
```

Comment:
Semver only looks for the common issues, we shouldn't rely on it to find all possible issues

Reply:
The problem is that before the changes, if the client code (let it be generic-ec-zkp or any other lib) had comparison like `Point<E> == O` where `O` is some type to be deduced by the compiler, compiler previously could do that, since point could only be compared with another point, but now it breaks compilation